### ISSUE TITLE
additionalPolicies seems broken on 1.5.0-alpha4

### ISSUE THREAD

**Issue by weaseal** [kubernetes/kops#1631]
Following this example: https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md

I pasted in exactly the "additionalPolicies" section from the above link while using 'kops edit cluster'.  After I exit and re-open 'kops edit cluster' the additionalPolicies section is missing entirely.

I also tried 'kops edit cluster'; 'kops update cluster --yes' to see if this would help, but the changes are still lost.

**Comment by yissachar** [comment]
I've just tested on 1.5.0-beta1 and it works fine for me. Here's what I did:

1. `kops edit cluster`
2. Move cursor down to `spec->channel` line
3. `o` to enter Insert Mode on next line
4. Paste the example code
5. `esc` to exit Insert Mode
6. `:wq` to save and quit
7. `kops edit` verify that the section still exists


YAML is fussy, most likely you somehow pasted at the wrong indent level.

**Comment by weaseal** [comment]
On 1.5.0-beta1 it outputs an error.  Here's the config:
```
kind: Cluster
metadata:
  creationTimestamp: "2017-01-19T23:33:56Z"
  name: redacted
spec:
  additionalPolicies:
    node: |
      [
        {
          "Effect": "Allow",
          "Action": ["dynamodb:*"],
          "Resource": ["*"]
        },
        {
          "Effect": "Allow",
          "Action": ["es:*"],
          "Resource": ["*"]
        }
      ]
  api:
    loadBalancer:
      type: Public
  channel: stable
  cloudProvider: aws
  configBase: s3://redacted
  etcdClusters:
  - etcdMembers:
    - instanceGroup: master-us-west-1b
      name: us-west-1b
    name: main
  - etcdMembers:
    - instanceGroup: master-us-west-1b
      name: us-west-1b
    name: events
  kubernetesApiAccess:
    - redacted/32
  kubernetesVersion: v1.5.2
  masterPublicName: redacted
  networkCIDR: 10.0.0.0/16
  networkID: vpc-0b03c56f
  networking:
    kubenet: {}
  nonMasqueradeCIDR: 100.64.0.0/10
  sshAccess:
  - redacted/32
  subnets:
  - cidr: 10.0.3.0/24
    name: k8s-subnet-us-west-1b
    type: Public
    zone: us-west-1b
  - cidr: 10.0.4.0/24
    name: k8s-subnet-us-west-1c
    type: Public
    zone: us-west-1c
  topology:
    dns:
      type: Public
    masters: public
    nodes: public
```

Followed by 'kops replace -f cluster.yaml' and then 'kops update --yes':
``` I0126 11:15:15.369751    8775 executor.go:91] Tasks: 71 done / 72 total; 1 can run
 W0126 11:15:15.563810    8775 executor.go:109] error running task "IAMInstanceProfileRole/redacted" (4m32s remaining to succeed): error creating IAMInstanceProfileRole: LimitExceeded: Cannot exceed quota for InstanceSessionsPerInstanceProfile: 1
	status code: 409, request id: c489f2ab-e3fb-11e6-bebf-c99b4f56e586
 I0126 11:15:15.563934    8775 executor.go:124] No progress made, sleeping before retrying 1 failed task(s)```

**Comment by yissachar** [comment]
I can reproduce this error by creating a cluster with `additionalPolicies`, deleting the cluster, and then attempting to re-create the cluster with `additionalPolicies`.

I guess something is not being properly cleaned up on cluster deletion - looking into it.

**Comment by yissachar** [comment]
I may have spoken too fast. I can't reliably reproduce this with the above steps, so it may not be related to deletion.

**Comment by chrislovecnm** [comment]
Can we get a -v 10 output? Also what are you quotas like? Check trustee advisor.

**Comment by weaseal** [comment]
So this was happening on an existing cluster, and then again on the same cluster after deleting and quickly recreating it, but it doesn't happen if I delete the cluster, and wait >60 minutes to recreate.

**Comment by chrislovecnm** [comment]
@weaseal can you run -v 10 with the next few runs?  I just `|& tee /tmp/kops.log`

**Comment by justinsb** [comment]
This was (hopefully) fixed by #1707.
